packages/fhir-converter/src/lib/handlebars-converter/handlebars-helpers.js

```diff
@@ -190,6 +194,11 @@ var getDateTime = function (dateTimeString) {
   if (timeZoneChar !== "") {
     var dateSections = ds.split(timeZoneChar);
     var dateTimeComposition = getDateTimeComposition(dateSections[0]);
+
+    if (parseInt(dateTimeComposition.year) < 1900) return "";
```

Comment:
I see we also have this below. Maybe just create a small function `isValidYear()` and call on both?